assets/src/dashboard/animations/animator/output.js

```diff
@@ -19,9 +19,20 @@
  */
 import PropTypes from 'prop-types';
 
-function WithAnimation({ id, style, children }) {
-  return (
-    <div id={id} style={style}>
+function WithAnimation({ id, style, animationStyle, useContainer, children }) {
+  return useContainer ? (
+    <div
+      style={{
+        clipPath: 'inset(0)',
```

Comment:
The use of `clipPath` here is simply clipping the container similar to adding `overflow: hidden`.